tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -61,7 +61,12 @@ <h2>Settings</h2>
     <paper-header-panel>
       <paper-toolbar id="toolbar" slot="header" class="header">
         <div id="toolbar-content" slot="top">
-          <div class="toolbar-title">[[brand]]</div>
+          <template is="dom-if" if="[[!homeHref]]">
+            <div class="toolbar-title">[[brand]]</div>
+          </template>
+          <template is="dom-if" if="[[homeHref]]">
+            <a href="[[homeHref]]" class="toolbar-title">[[brand]]</a>
```

Comment:
Especially given we do not add `rel="nooopener"`, we should sanitize the input to make sure it is relative. WDYT?

With the change, we probably should change `homeHref` to `homePath`.

Reply:
I think only the admin will be able to set this value so I don't think we need to worry about malicious URLs.

I also don't think we need to restrict this value to relative paths.

Do you have an example scenario that concerns you?

Reply:
Technically, there are more ways to abuse it.

Although impractical:
- MITM (since TB is often hosted without HTTPS)
- Chrome Extension (admit that you can already can pwn so many websites when you are on extension)
- Any JavaScript running on page
  - `document.querySelector('tf-tensorboard').homeUrl = 'javascript:alert(...)'.`
  - note that you do not have resin on OSS
  - note that CSP header can be stripped out by random proxy

Reply:
Thanks for helping me take care of this with your commits. If the sanitization ends up being too restrictive in the future we can always reconsider. In the meantime it should suit our needs.